packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/nodeexpress/integrationsummary.md

```diff
@@ -0,0 +1,5 @@
+<div class="full">
+
+![Sequence diagram illustrating the password-optional sign-in use case.](/img/pwd-optional/pwd-optional-sign-in-summary.png)
```

Comment:
A few notes on the diagram based on our discussions about saying who is doing what step

Step 1-2 should be called something like "The user submits their username" or "The user starts the flow and submits their username". 
Step 3 perhaps "The user verifies their identity with Identity Engine using the email authenticator"
Step 4 should be "The user completes the sign-in flow successfully"

"Display a page that supports verifying identity with the email authenticator" doesn't say anything. What is actually happening here? Are we querying the user for their email address to send the email to? Are we displaying a prompt for the user to open their email? Are we displaying a choice of authenticators (showing only the email authetnicator) for the user to select from? I think you need t be more precise.